src/Oauth2CredentialManagers/CacheStore.php

```diff
@@ -8,81 +8,32 @@
 use Webfox\Xero\Oauth2Provider;
 use Webfox\Xero\OauthCredentialManager;
 
-class CacheStore implements OauthCredentialManager
+class CacheStore extends BaseCredentialManager implements OauthCredentialManager
 {
-    protected string $cacheKey = 'xero_oauth';
+    protected string $cacheKey = 'xero_state';
 
     public function __construct(protected Repository $cache, protected Store $session, protected Oauth2Provider $oauthProvider)
     {
     }
 
-    public function getAccessToken(): string
-    {
-        return $this->data('token');
-    }
-
-    public function getRefreshToken(): string
-    {
-        return $this->data('refresh_token');
-    }
-
-    public function getTenants(): ?array
-    {
-        return $this->data('tenants');
-    }
-
-    public function getTenantId(int $tenant = 0): string
-    {
-        if (! isset($this->data('tenants')[$tenant])) {
-            throw new \Exception('No such tenant exists');
-        }
-
-        return $this->data('tenants')[$tenant]['Id'];
-    }
-
-    public function getExpires(): int
-    {
-        return $this->data('expires');
-    }
-
     public function getState(): string
     {
-        return $this->session->get($this->cacheKey);
+        return $this->session->get('xero_oauth2_state');
```

Comment:
I've changed the get state session key to be consistent with FileStore, I didn't think this would cause too many issues - sessions would be wiped after a while anyway - do you see any issues with this @hailwood

Reply:
I don't see any issue with this :)